src/pages/NewChairConsole.js

```diff
@@ -38,13 +39,39 @@ const getElectionDate = (election) => {
 
 const Records = (props) => {
   //NOTE: currentPage is not zero-indexed
-  const {openModal, currentPage, tableSize, applyTextHover, removeTextHover} = props;
+  const {openModal, currentPage, tableSize, applyTextHover, removeTextHover, openConfirmation} = props;
   const startIndex = (currentPage - 1) * tableSize;
   const endIndex = currentPage * tableSize; //NOTE: endIndex is exclusive, not inclusive
   const visibleWindow = props.filteredList.slice(startIndex, endIndex);
   const dataIDTextStyle = Styles.TABLE.DATA_REQUEST_TEXT;
   const recordTextStyle = Styles.TABLE.RECORD_TEXT;
 
+  const createActionButton = (electionInfo, index) => {
+    const name = "cell-button hover-color";
+    const e = electionInfo.election;
+    const dar = electionInfo.dar;
+    if (e !== null) {
```

Comment:
I think this should be the last bug left to be ironed out. After this PR should be good to go.

In javascript variables can be ```null``` or ```undefined``` however these values are not equal to each other. So if ```e = undefined``` then the if statement will return true since ```undefined !== null```. A good way to deal with this is to use lodash's isNil method (which is imported at the top of the file) which will treat undefined and null as equivalent values (since both in a sense represent an absence of value).

Reply:
awesome! I will make that change